src/D2L.Bmx/LoginHandler.cs

```diff
@@ -5,14 +5,21 @@ OktaAuthenticator oktaAuth
 ) {
 	public async Task HandleAsync(
 		string? org,
-		string? user
+		string? user,
+		bool experimentalBypassBrowserSecurity
 	) {
 		if( !File.Exists( BmxPaths.CONFIG_FILE_NAME ) ) {
 			throw new BmxException(
 				"BMX global config file not found. Okta sessions will not be saved. Please run `bmx configure` first."
 			);
 		}
-		await oktaAuth.AuthenticateAsync( org, user, nonInteractive: false, ignoreCache: true );
+		await oktaAuth.AuthenticateAsync(
+			org,
+			user,
+			nonInteractive: false,
+			ignoreCache: true,
+			experimentalBypassBrowserSecurity: experimentalBypassBrowserSecurity
```

Comment:
nit: the CLI flag starts with `--experimental-` but the internal variable/parameter names can maybe be shortened to not include "experimental" everywhere?